### ISSUE TITLE
Feature request: system volume popup

### ISSUE THREAD

**Issue by aredfan** [LostArtefacts/TRX#635]
Hello.  I thought it might be a good idea to mention that there's no system volume popup on Tomb1Main when adjusting or muting the volume.  The same thing happens on TombATI so it's not a T1M-only issue.  The system volume popup is typically visible for all games on Windows.

For example, here's a screenshot from TR2:

![Screenshot](https://user-images.githubusercontent.com/94776985/194750078-a63af574-e340-4ac9-a85f-327ca9d1a44c.png)

**Update:** This is what the volume popup looks like in Windows 11

![Screenshot (1)](https://user-images.githubusercontent.com/94776985/197777748-894b4c4c-58df-409d-a22d-c570b6d5fe22.png)

**Comment by Richard-L** [comment]
The one you're showing is a Win10 overlay. I don't have it on Win7, and not sure about Win8.

**Comment by walkawayy** [comment]
@aredfan I was researching a fix for different issues and may have found a fix for this?! Granted it seems to require an Nvidia GPU (idk if AMD has an equivalent) and I'm not sure if all Nvidia GPUs have this setting. It seems to also fix issues with a few black flickers especially when I tried to take screenshots with the Windows snipping tool.

I set `Vulkan/OpenGL present method: Prefer layered on DXGI Swapchain`.

**Note however if you do have an HDR display, I think it might interfere with HDR.**

![image](https://github.com/user-attachments/assets/a7dabfa8-2aa7-4940-bddf-db33abe888ef)

**Comment by aredfan** [comment]
@walkawayy I'm staggered because this solution actually fixed the issue for me. Thank you for researching it. :D 

@rr- This method also fixed the flickering mentioned in #1388.

![Screenshot (1722)](https://github.com/user-attachments/assets/4bd66106-e829-476d-b168-c8b457caf3c4)

**Comment by walkawayy** [comment]
Yes it seems like there's some OpenGL behind the scenes stuff going on we don't understand. I got a new monitor recently with HDR which is why I was even looking into this option. Unfortunately, it looks like this option doesn't play nice with HDR from what I can tell. But for anyone else it seems like a good solution! Maybe we can fix this in the future natively without having to mess with this option since I honestly am not sure if it will mess with other games or functionality. I'll have to research it more in the future.

**Comment by aredfan** [comment]
I played a non-TR game yesterday and the Windows volume pop-up wasn't visible in said game.

At the time of writing this issue three years ago, I initially thought it was a TombATI specific bug, but perhaps I was mistaken in making that assumption.
